tox.ini

```
@@ -1,18 +1,19 @@
 [tox]
-min_version = 4.22
+min_version = 4.32
+skipmissing_interpreters = true
 
 envlist =
-    python{3.9,3.10}-django4.2-wagtail{6.3}
-    python{3.10,3.11}-django5.1-wagtail{7.0,7.1}
-    python{3.12,3.13}-django5.2-wagtail{7.0,7.1}
+    python{3.10,3.11,3.12}-django4.2-wagtail{7.0,7.1,7.2}
+    python{3.10,3.11,3.12,3.13}-django{5.1,5.2}-wagtail{7.0,7.1,7.2}
+    python3.14-django5.2-wagtail{7.0,7.1,7.2}
```

how about

```suggestion
    python{3.10,3.11}-django4.2-wagtail{7.1}
    python{3.12,3.13}-django{5.1,5.2}-wagtail{7.0,7.2}
    python3.14-django5.2-wagtail{7.2}
```

smaller test matrix, but still covers what we need